cmd/katalyst-controller/app/options/cnr.go

```diff
@@ -0,0 +1,56 @@
+/*
```

Comment:
Does this file overlap with the `controllers` parameter?
https://github.com/kubewharf/katalyst-core/blob/a3bef5bd1e5cfb0f5ccf6bee32863d7e20840788/cmd/katalyst-controller/app/options/controllerbase.go#L83